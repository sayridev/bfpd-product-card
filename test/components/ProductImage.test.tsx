import React from 'react';
import rendered from 'react-test-renderer';
import ProductCard, { ProductImage } from '../../src/components';
import { product2 } from '../data/products';
describe('ProductImage', () => {
  test('Debe mostrar la imagen personlizada', () => {
    const wrapper = rendered.create(<ProductImage img="htpp:image.com" />);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
  test('Debe mostrar la imagen del producto ', () => {
    const wrapper = rendered.create(
      <ProductCard product={product2}>{() => <ProductImage />}</ProductCard>
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
