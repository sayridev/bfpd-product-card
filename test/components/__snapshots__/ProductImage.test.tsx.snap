// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductImage Debe mostrar la imagen del producto  1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="Product Image"
    className="productImg undefined"
    src="./coffee-mug2.png"
  />
</div>
`;

exports[`ProductImage Debe mostrar la imagen personlizada 1`] = `
<img
  alt="Product Image"
  className="productImg undefined"
  src="htpp:image.com"
/>
`;
